govcd/metadata.go

```diff
@@ -171,22 +166,50 @@ func (vdc *Vdc) AddMetadataEntry(typedValue, key, value string) error {
 	return nil
 }
 
+// AddMetadataEntry adds VDC metadata typedValue and key/value pair provided as input
+// and waits for the task to finish.
+// Deprecated: Use AdminVdc.AddMetadataEntryWithVisibility instead
+func (adminVdc *AdminVdc) AddMetadataEntry(typedValue, key, value string) error {
```

Comment:
I added this function, although is already deprecated, just to be able to use interfaces in the consumer side (Terraform Provider in this case), which generalizes the usage of metadata and avoids code duplication.